samples/Commerce/Commerce.UI/Views/ProductsPage.xaml

```diff
@@ -90,21 +90,21 @@
 									Orientation="Horizontal"
 									utu:AutoLayout.CounterAlignment="Stretch"
 									utu:AutoLayout.PrimaryLength="20">
-						<TextBlock Foreground="{StaticResource MaterialOnBackgroundBrush}"
+						<TextBlock Foreground="{StaticResource OnBackgroundBrush}"
 								   x:Uid="Page_5254 products found!"
 								   Text="5254 products found!"
 								   utu:AutoLayout.CounterAlignment="Center"
-								   Style="{StaticResource MaterialBody2}" />
+								   Style="{StaticResource BodySmall}" />
 						<HyperlinkButton Content="Filters"
 										 AutomationProperties.AutomationId="Filters"
 										 x:Uid="Page_Filters"
 										 utu:AutoLayout.CounterAlignment="Center"
-										 Style="{StaticResource MaterialHyperlinkButtonStyle}"
+										 Style="{StaticResource HyperlinkButtonStyle}"
 										 uen:Navigation.Request="!Filter"
 										 uen:Navigation.Data="{Binding Filter.Value, Mode=TwoWay}" />
 					</utu:AutoLayout>
 				</utu:AutoLayout>
-				<uer:FeedView Style="{StaticResource MaterialCompositeFeedViewStyle}"
+				<uer:FeedView Style="{StaticResource CompositeFeedViewStyle}"
```

Comment:
This is a custom style defined in the app, and I believe it still has a Material prefix at this point